pkg/bbgo/exit_trailing_stop.go

```diff
@@ -16,6 +16,9 @@ type TrailingStop2 struct {
 	// CallbackRate is the callback rate from the previous high price
 	CallbackRate fixedpoint.Value `json:"callbackRate,omitempty"`
 
+	// CallbackProfitRate is the callback rate from the previous high profit
+	CallbackProfitRate fixedpoint.Value `json:"callbackProfitRate,omitempty"`
```

Comment:
what's the difference between CallbackRate? previous high price also equals to previous high profit